test/unit/force/org/clone.test.ts

```diff
@@ -77,9 +83,6 @@ describe('org:clone', () => {
   };
 
   // stubs
-  let uxTableStub: sinon.SinonStub;
-  let uxStyledHeaderStub: sinon.SinonStub;
-  let uxLogStub: sinon.SinonStub;
   let requestStub: sinon.SinonStub;
   // let aliasSetStub: sinon.SinonSpy;
```

Comment:
I'll work on that but I think the recommendation here should be to have testable/stubbable methods in your command class for that sort of thing.

So if I command uses `Config` to set a config var, then instead of subbing `Config` plugin providers should have a method on their command (e.g. `setConfigVar`) that they can stub instead

That way they don't have to concern themselves with the inner-workings of a library they don't own